apps/src/sites/studio/pages/maker/setup.js

```diff
@@ -111,15 +121,15 @@ const BoardSetupStatus = React.createClass({
      */
     function forEachLedInSequence(func, delay) {
       return new Promise(resolve => {
-        const leds = bc.prewiredComponents.colorLeds;
+        const leds = board.prewiredComponents_.colorLeds;
```

Comment:
I very much like the idea of giving the board a celebrate function that this method can incorporate into its chain in lieu of the `buzzer.play` and `forEachLedInSequence` calls.